src/variables.css

```diff
@@ -58,9 +58,11 @@
     --font-caption: var(--size-4) / 1.42 var(--font-ui);
 
     /* Shadows */
-    --shadow-1: 0 1px 3px #0003, 0 0 0 1px #0001;
-    --shadow-2: 0 2px 6px #0003, 0 0 0 1px #0001;
-    --shadow-3: 0 4px 12px #0003, 0 0 0 1px #0001;
+    --shadow-line: 0 0 1px #0001;
```

Comment:
Ok, I now understand the problem here. `0 0 0 1px #0001` is the value removed from the shadows, and `0 0 1px #0001` the value added to `--shadow-line`. Note the zero that was removed. 

The problem is that they are not the same value, and sadly, `drop-shadow` does not support the 4 numbers syntax. I see two options here:

- Increase the darkness of the color in shadow-line. For example `0 0 1px #0006` to compensate the lack of the spread value, that makes the color more light.
- Do not use the shadow-line variable in the tooltip and use a "hard typed" value.

Reply:
I have tried the first option and work fine. Please, check it.